src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/ExceptionRecordFinalizerTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.DISPLAY_WARNINGS;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.OCR_DATA_VALIDATION_WARNINGS;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.YesNoFieldValues.NO;
+
+class ExceptionRecordFinalizerTest {
+
+    private static final long CASE_ID = 100L;
+    private ExceptionRecordFinalizer exceptionRecordFinalizer;
+
+    @BeforeEach
+    void setUp() {
+        exceptionRecordFinalizer = new ExceptionRecordFinalizer();
+    }
+
+    @Test
+    void should_add_new_values() {
+        // given
+        Map<String, Object> originalValues = new HashMap<>();
+        originalValues.put("field1", "value1");
+
+        // when
+        Map<String,Object> res = exceptionRecordFinalizer.finalizeExceptionRecord(originalValues, CASE_ID);
+
+        // then
+        assertThat(res.get("field1")).isEqualTo("value1");
+        assertThat(res.get(ExceptionRecordFields.CASE_REFERENCE)).isEqualTo(Long.toString(CASE_ID));
+        assertThat(res.get(DISPLAY_WARNINGS)).isEqualTo(NO);
+        assertThat(res.get(OCR_DATA_VALIDATION_WARNINGS)).isEqualTo(emptyList());
```

Comment:
Worth adding an assertion that there are no more keys than those 4.

Reply:
Will do